src/bap/arm.py

```diff
@@ -6,239 +6,873 @@
-class MUL(Mult) : pass
-class SMLABB(Mult) : pass
-class SMLAD(Mult) : pass
-class SMLAL(Mult) : pass
-class SMLALBT(Mult) : pass
-class SMLAWB(Mult) : pass
-class SMUAD(Mult) : pass
-class SMULBB(Mult) : pass
-class SMULL(Mult) : pass
-class SMULTB(Mult) : pass
-class UMLAL(Mult) : pass
-class UMULL(Mult) : pass
-
-class LDMDA(Mem) : pass
-class LDMDA_UPD(Mem) : pass
-class LDMDB(Mem) : pass
-class LDMDB_UPD(Mem) : pass
-class LDMIA(Mem) : pass
-class LDMIA_UPD(Mem) : pass
-class LDMIB(Mem) : pass
-class LDMIB_UPD(Mem) : pass
-class STMDA(Mem) : pass
-class STMDA_UPD(Mem) : pass
-class STMDB(Mem) : pass
-class STMDB_UPD(Mem) : pass
-class STMIA(Mem) : pass
-class STMIA_UPD(Mem) : pass
-class STMIB(Mem) : pass
-class STMIB_UPD(Mem) : pass
-class LDRBT_POST_IMM(Mem) : pass
-class LDRBT_POST_REG(Mem) : pass
-class LDRB_POST_IMM(Mem) : pass
-class LDRB_POST_REG(Mem) : pass
-class LDRB_PRE_IMM(Mem) : pass
-class LDRB_PRE_REG(Mem) : pass
-class LDRBi12(Mem) : pass
-class LDRBrs(Mem) : pass
-class LDRD(Mem) : pass
-class LDRD_POST(Mem) : pass
-class LDRD_PRE(Mem) : pass
-class LDREX(Mem) : pass
-class LDREXB(Mem) : pass
-class LDREXD(Mem) : pass
-class LDREXH(Mem) : pass
-class LDRH(Mem) : pass
-class LDRHTr(Mem) : pass
-class LDRH_POST(Mem) : pass
-class LDRH_PRE(Mem) : pass
-class LDRSB(Mem) : pass
-class LDRSBTr(Mem) : pass
-class LDRSB_POST(Mem) : pass
-class LDRSB_PRE(Mem) : pass
-class LDRSH(Mem) : pass
-class LDRSHTi(Mem) : pass
-class LDRSHTr(Mem) : pass
-class LDRSH_POST(Mem) : pass
-class LDRSH_PRE(Mem) : pass
-class LDRT_POST_REG(Mem) : pass
-class LDR_POST_IMM(Mem) : pass
-class LDR_POST_REG(Mem) : pass
-class LDR_PRE_IMM(Mem) : pass
-class LDR_PRE_REG(Mem) : pass
-class LDRi12(Mem) : pass
-class LDRrs(Mem) : pass
-class STRBT_POST_IMM(Mem) : pass
-class STRBT_POST_REG(Mem) : pass
-class STRB_POST_IMM(Mem) : pass
-class STRB_POST_REG(Mem) : pass
-class STRB_PRE_IMM(Mem) : pass
-class STRB_PRE_REG(Mem) : pass
-class STRBi12(Mem) : pass
-class STRBrs(Mem) : pass
-class STRD(Mem) : pass
-class STRD_POST(Mem) : pass
-class STRD_PRE(Mem) : pass
-class STREX(Mem) : pass
-class STREXB(Mem) : pass
-class STREXD(Mem) : pass
-class STREXH(Mem) : pass
-class STRH(Mem) : pass
-class STRHTr(Mem) : pass
-class STRH_POST(Mem) : pass
-class STRH_PRE(Mem) : pass
-class STRT_POST_REG(Mem) : pass
-class STR_POST_IMM(Mem) : pass
-class STR_POST_REG(Mem) : pass
-class STR_PRE_IMM(Mem) : pass
-class STR_PRE_REG(Mem) : pass
-class STRi12(Mem) : pass
-class STRrs(Mem) : pass
-
-class BL(Branch) : pass
-class BLX(Branch) : pass
-class BLX_pred(Branch) : pass
-class BLXi(Branch) : pass
-class BL_pred(Branch) : pass
-class BX(Branch) : pass
-class BX_RET(Branch) : pass
-class BX_pred(Branch) : pass
-class Bcc(Branch) : pass
-
-class CPS2p(Special) : pass
-class DMB(Special) : pass
-class DSB(Special) : pass
-class HINT(Special) : pass
-class MRS(Special) : pass
-class MSR(Special) : pass
-class PLDi12(Special) : pass
-class SVC(Special) : pass
+
```

Comment:
This renders the module totally unreadable. The whole idea is that you can quickly see different variants of the ADT.